bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java

```diff
@@ -462,8 +462,11 @@ private SOAPMessage readSoapResponse(String soapActionHeader, SOAPMessage reques
             // Check for (auth-) error
             if (slResponse.getStatusCode() == 401) {
                 logger.error(
```

Comment:
Please convert this to a WARN instead of ERROR.

Reply:
:+1: This wasn't mine, but I changed it.